python/pylibcudf/pylibcudf/tests/io/test_types.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import gc
+import weakref
+
+import pyarrow as pa
+
+import pylibcudf as plc
+
+
+def test_gc_with_table_and_column_input_metadata():
+    class Foo(plc.io.types.TableInputMetadata):
+        def __del__(self):
+            pass
+
```

Comment:
```suggestion
```

Reply:
We only need the __del__ in this test, otherwise I'd need to add it to the class I think